docs/guides/modules/permissions-authentication/pages/sso-overview.adoc

```diff
@@ -32,7 +32,24 @@ Using SSO for authentication can result in improved security and user experience
 * Adding and deleting users.
 * Managing user permissions.
 
-**For a member of an organization using SSO for authentication:** The process of authentication to access applications is simplified. Rather than authenticating separately for each application, SAML SSO enables the user to confirm their identity _once_ with an IdP and that authentication is communicated to the applications (such as CircleCI) that the user has access to. The user may then access these applications for the duration of their session.
+**For a member of an organization using SSO for authentication:** The process of authentication to access applications is simplified. Rather than authenticating separately for each application, SAML SSO enables the user to confirm their identity _once_ with an IdP. This authentication is then communicated to all applications (such as CircleCI) that the user has access to. The user may then access these applications for the duration of their session.
+
+== Personal API tokens with SSO
+
+When your organization uses SSO, Personal API tokens require SSO authentication to access your organization's resources.
+
+To use Personal API tokens with SSO-protected resources:
+
+. In the link:https://app.circleci.com/home/[CircleCI web app], select your profile from the upper right corner, then select **User Settings**.
+. Select *Personal API Tokens* from the sidebar.
```

Comment:
Use the AsciiDoc menu macro:

Select menu:Personal API Tokens[] from the sidebar.